filebeat/docs/index.asciidoc

```diff
@@ -21,6 +21,10 @@ include::../../libbeat/docs/shared-directory-layout.asciidoc[]
 
 include::./configuring-howto.asciidoc[]
 
+include::../../libbeat/docs/yaml.asciidoc[]
+
+include::../../libbeat/docs/regexp.asciidoc[]
+
 include::./filebeat-filtering.asciidoc[]
 
 include::./multiline.asciidoc[]
```

Comment:
Not 100% sure about the ordering here, as somehow further up means more important. I see the logic that regexp sohuld be understood to use mutiline, but I think people will first look for multiline and then jump to regexp. Same for yaml?